spec/factories/sign_offs.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :sign_off do
+    provider
+    academic_cycle
+    user
+    sign_off_type { "MyString" }
+
+    trait :performance_profile do
+      sign_off_type { "performance_profile" }
+    end
+
+    trait :census do
+      sign_off_type { "census" }
+    end
```

Comment:
Don't think it is needed, it's out of the box.

default it to 
```suggestion
    performance_profile
```

https://thoughtbot.github.io/factory_bot/traits/enum.html

Reply:
Nice! 👍